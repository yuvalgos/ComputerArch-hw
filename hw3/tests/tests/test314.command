./dflow_calc tests/test314.dat tests/test314.in d35 p25 p18 d17 p138 d25 p148 p182 p178 d36 d115 p77 p128 p163 d138 p191 p13 d170 d151 d10 p12 d105 p120 d30 d127 d51 d70 d1 d21 d22 p187 p29 p77 d16 d50 d105 p142 d127 p110 p48 p81 p59 p33 d3 d196 p137 d15 d135 d128 p25 d185 d126 d7 d88 p7 d142 d124 d14 d88 d10 p88 d157 p15 d23 p97 p168 d97 p95 p141 d31 p35 p94 p74 p50 p2 p110 p137 d16 d157 p86 p125 p174 d192 p22 d57 d170 d109 p108 p103 d186 d49 d137 p164 p116 d41 d143 d155 d5 p155 d35 d64 d13 p101 p173 d30 p175 p127 d89 d17 p107 p198 p43 p137 d139 p101 d170 d141 p37 d48 p53 p159 p109 p58 p175 p88 p42 p105 d128 p24 p137 d87 p93 d45 p174