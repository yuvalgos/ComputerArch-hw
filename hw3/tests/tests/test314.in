7 17 3 19
6 16 20 20
8 17 28 10
8 25 27 29
4 6 13 28
4 27 4 28
4 11 18 19
4 13 5 21
4 6 24 3
8 21 9 15
1 16 19 15
0 0 28 23
6 8 14 26
7 26 12 24
0 11 15 17
3 5 10 20
5 22 25 31
5 28 10 3
6 20 21 17
6 24 13 1
3 9 29 12
1 13 3 20
8 29 4 9
4 21 19 17
7 11 4 20
2 3 15 28
7 16 19 0
7 15 24 24
0 11 7 4
2 6 23 6
5 23 31 8
3 12 14 6
5 9 17 0
0 7 31 25
3 23 22 10
8 9 24 0
7 13 29 15
1 11 13 22
4 22 9 23
3 15 5 16
7 8 22 29
1 2 19 31
3 1 23 17
4 30 8 18
2 28 19 14
0 16 3 27
4 23 15 19
7 29 20 14
7 1 0 13
3 27 2 1
6 14 25 13
7 29 29 9
3 4 29 12
7 2 4 23
2 1 12 28
4 13 24 28
8 6 14 19
6 13 22 23
4 17 17 14
6 6 31 16
3 3 28 6
2 1 2 17
4 13 29 16
7 21 15 13
4 23 12 28
3 8 4 17
2 24 0 13
8 1 19 25
0 19 16 30
1 5 26 31
1 23 25 16
6 21 14 26
5 26 3 5
6 5 10 26
0 11 9 27
5 2 0 2
4 11 29 22
0 0 5 0
8 23 31 24
7 26 6 0
3 14 26 13
3 22 4 28
3 23 3 25
0 2 9 17
3 27 15 6
7 10 25 6
3 31 4 11
3 18 2 20
0 0 24 1
2 15 12 25
6 13 1 2
1 4 4 9
4 26 23 15
2 10 30 10
5 7 4 11
8 20 29 16
6 6 22 15
0 10 14 26
1 4 8 10
3 8 3 11
1 22 2 0
7 26 8 16
3 24 5 16
1 0 18 17
0 8 1 14
2 25 8 21
2 31 29 5
7 31 15 25
7 25 11 28
1 29 0 7
6 8 15 1
5 26 25 3
8 27 0 8
1 10 16 31
8 13 0 5
1 26 3 10
6 11 3 21
6 8 1 26
5 24 21 24
7 26 24 3
8 4 25 5
5 29 29 15
1 25 0 23
1 18 1 18
1 6 27 26
6 6 21 2
1 15 27 31
2 30 29 28
7 25 31 0
7 5 19 29
6 23 2 30
3 25 29 22
7 16 1 27
6 25 8 15
3 16 15 0
2 6 18 2
3 19 15 0
3 17 8 14
1 6 3 20
1 25 29 20
2 21 13 14
2 31 26 29
0 0 2 13
0 30 28 22
1 0 5 26
2 27 3 1
3 23 30 24
3 30 7 10
0 20 28 0
7 17 8 12
5 4 27 30
6 14 10 8
8 6 12 3
2 0 3 4
8 1 27 10
1 16 20 20
0 17 25 2
0 7 21 2
7 8 0 13
2 15 5 27
0 13 1 5
5 5 2 31
3 3 27 16
0 21 16 27
6 1 16 14
3 17 24 9
6 27 12 18
5 25 10 2
0 14 24 16
0 16 1 7
7 26 4 8
0 4 6 28
4 31 22 7
5 11 30 19
0 13 12 28
0 25 12 6
3 1 11 6
6 10 29 8
1 23 24 11
7 9 8 30
3 15 21 24
8 27 24 27
5 20 27 31
8 15 10 12
3 0 25 18
3 1 9 8
3 29 24 27
2 20 30 21
7 12 26 1
0 20 16 24
7 3 18 22
3 7 3 5
0 20 30 12
6 17 14 22
3 13 14 15
2 14 10 31
0 18 12 29
2 3 9 18
1 22 30 8
8 13 3 26
