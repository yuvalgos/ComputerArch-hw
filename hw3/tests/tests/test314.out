Reading the opcodes latency info from tests/test314.dat ... Got latency for 9 opcodes
Reading the program file tests/test314.in ... Found 200 instructions
getProgDepth()==250
getInstDeps(35)=={19,11}
getDepDepth(25)==13
getDepDepth(18)==23
getInstDeps(17)=={-1,-1}
getDepDepth(138)==101
getInstDeps(25)=={-1,17}
getDepDepth(148)==144
getDepDepth(182)==147
getDepDepth(178)==152
getInstDeps(36)=={22,27}
getInstDeps(115)=={60,113}
getDepDepth(77)==74
getDepDepth(128)==128
getDepDepth(163)==149
getInstDeps(138)=={60,95}
getDepDepth(191)==150
getDepDepth(13)==0
getInstDeps(170)=={150,158}
getInstDeps(151)=={113,117}
getInstDeps(10)=={-1,-1}
getDepDepth(12)==0
getInstDeps(105)=={104,71}
getDepDepth(120)==108
getInstDeps(30)=={-1,12}
getInstDeps(127)=={121,44}
getInstDeps(51)=={47,35}
getInstDeps(70)=={3,45}
getInstDeps(1)=={-1,-1}
getInstDeps(21)=={-1,18}
getInstDeps(22)=={-1,20}
getDepDepth(187)==151
getDepDepth(29)==11
getDepDepth(77)==74
getInstDeps(16)=={3,-1}
getInstDeps(50)=={3,36}
getInstDeps(105)=={104,71}
getDepDepth(142)==129
getInstDeps(127)=={121,44}
getDepDepth(110)==46
getDepDepth(48)==66
getDepDepth(81)==85
getDepDepth(59)==20
getDepDepth(33)==23
getInstDeps(3)=={-1,-1}
getInstDeps(196)=={188,186}
getDepDepth(137)==151
getInstDeps(15)=={-1,-1}
getInstDeps(135)=={123,83}
getInstDeps(128)=={107,103}
getDepDepth(25)==13
getInstDeps(185)=={179,158}
getInstDeps(126)=={112,107}
getInstDeps(7)=={-1,-1}
getInstDeps(88)=={66,67}
getDepDepth(7)==0
getInstDeps(142)=={83,114}
getInstDeps(124)=={112,119}
getInstDeps(14)=={-1,2}
getInstDeps(88)=={66,67}
getInstDeps(10)=={-1,-1}
getDepDepth(88)==94
getInstDeps(157)=={140,83}
getDepDepth(15)==0
getInstDeps(23)=={-1,2}
getDepDepth(97)==102
getDepDepth(168)==149
getInstDeps(97)=={80,92}
getDepDepth(95)==78
getDepDepth(141)==132
getInstDeps(31)=={-1,29}
getDepDepth(35)==28
getDepDepth(94)==89
getDepDepth(74)==92
getDepDepth(50)==66
getDepDepth(2)==0
getDepDepth(110)==46
getDepDepth(137)==151
getInstDeps(16)=={3,-1}
getInstDeps(157)=={140,83}
getDepDepth(86)==89
getDepDepth(125)==100
getDepDepth(174)==23
getInstDeps(192)=={147,188}
getDepDepth(22)==7
getInstDeps(57)=={38,46}
getInstDeps(170)=={150,158}
getInstDeps(109)=={103,94}
getDepDepth(108)==89
getDepDepth(103)==103
getInstDeps(186)=={118,181}
getInstDeps(49)=={41,48}
getInstDeps(137)=={117,80}
getDepDepth(164)==168
getDepDepth(116)==100
getInstDeps(41)=={-1,-1}
getInstDeps(143)=={44,100}
getInstDeps(155)=={148,148}
getInstDeps(5)=={-1,-1}
getDepDepth(155)==146
getInstDeps(35)=={19,11}
getInstDeps(64)=={31,44}
getInstDeps(13)=={-1,-1}
getDepDepth(101)==96
getDepDepth(173)==139
getInstDeps(30)=={-1,12}
getDepDepth(175)==67
getDepDepth(127)==122
getInstDeps(89)=={31,3}
getInstDeps(17)=={-1,-1}
getDepDepth(107)==109
getDepDepth(198)==175
getDepDepth(43)==81
getDepDepth(137)==151
getInstDeps(139)=={121,95}
getDepDepth(101)==96
getInstDeps(170)=={150,158}
getInstDeps(141)=={119,121}
getDepDepth(37)==66
getInstDeps(48)=={11,36}
getDepDepth(53)==85
getDepDepth(159)==141
getDepDepth(109)==106
getDepDepth(58)==83
getDepDepth(175)==67
getDepDepth(88)==94
getDepDepth(42)==43
getDepDepth(105)==104
getInstDeps(128)=={107,103}
getDepDepth(24)==40
getDepDepth(137)==151
getInstDeps(87)=={83,18}
getDepDepth(93)==92
getInstDeps(45)=={25,5}
getDepDepth(174)==23
